manifests/server/config.pp

```diff
@@ -1,11 +1,27 @@
 class kafka::server::config {
 
-  file { '/etc/init/kafka.conf':
-    ensure => 'file',
-    owner  => 'root',
-    group  => 'root',
-    mode   => '0444',
-    content => template('kafka/kafka.conf.erb'),
+  if $::lsbdistcodename == 'trusty' {
+    file { '/etc/init/kafka.conf':
+      ensure => 'file',
+      owner  => 'root',
+      group  => 'root',
+      mode   => '0444',
+      content => template('kafka/kafka.conf.erb'),
+    }
+  } else {
+    file { '/etc/init/pre-start.sh':
+      ensure => 'file',
+      owner  => 'root',
+      group  => 'root',
+      mode   => '0444',
+      content => template('kafka/pre-start.sh.erb'),
```

Comment:
since we're not using any embedded ruby in this script, i dont think we need to use a template here, should just use `source => `